Project.toml

```diff
@@ -0,0 +1,14 @@
+desc = "Allows Julia function calls to be temporarily overloaded for purpose of testing"
+keywords = ["testing", "mocking"]
+author = ["Curtis Vogt"]
+license = "MIT"
+name = "Mocking"
+uuid = "78c3b35d-d492-501b-9361-3d52fe80e533"
+version = "0.5.3"
+
+[deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
```

Comment:
Shouldn't be necessary but currently is. See https://github.com/JuliaLang/Pkg.jl/issues/427